handler/handler.go

```diff
@@ -47,3 +54,15 @@ func (q *NatsQueue) Queue(req *queue.Request) error {
 
 	return err
 }
+
+func (q *NatsQueue) reconnectClient(clientID string, clusterID string) nats.ConnHandler {
+	return func(c *nats.Conn) {
+		nc, err := stan.Connect(clusterID, clientID, stan.NatsConn(c))
+		if err != nil {
+			log.Println("Failed to reconnect to NATS stream")
```

Comment:
Does it make sense to bundle up a single error / log statement here with `log.Printf()`?

Reply:
Yes that makes more sense.